db/migrate/20151435234635_create_container_volumes.rb

```diff
@@ -0,0 +1,49 @@
+class CreateContainerVolumes < ActiveRecord::Migration
+  def up
+    create_table :container_volumes do |t|
+      t.belongs_to :container_group, :type => :bigint
+      t.string     :name
+      t.string     :host_path
+      t.string     :medium_type
+      t.string     :pd_name
+      t.string     :gce_fs_type
+      t.integer    :gce_partition
+      t.boolean    :gce_read_only
+      t.string     :aws_volume_id
+      t.string     :aws_fs_type
+      t.string     :aws_partition
+      t.boolean    :aws_read_only
+      t.string     :git_repository
+      t.string     :git_revision
+      t.string     :secret
+      t.string     :nfs_server
+      t.string     :nfs_path
+      t.boolean    :nfs_read_only
+      t.string     :target_portal
+      t.string     :iqn
+      t.integer    :lun
+      t.string     :iscsi_fs_type
+      t.boolean    :iscsi_read_only
+      t.string     :endpoint_name
+      t.string     :glusterfs_path
+      t.boolean    :glusterfs_read_only
+      t.string     :claim_name
+      t.boolean    :claim_read_only
+      t.string     :ceph_monitors
+      t.string     :image
+      t.string     :pool
+      t.string     :rados_user
+      t.string     :keyring
+      t.string     :secret_ref
+      t.string     :rbd_fs_type
+      t.boolean    :rbd_read_only
+      t.string     :cinder_volume_id
+      t.string     :cinder_fs_type
+      t.boolean    :cinder_read_only
```

Comment:
I guess what I always come back to is "what is the use case for this data".  If there's a use case, you need to collect it.

Ideally it would be named in a way that's cross-provider (so maybe remove the cinder/glusterfs  specfic distinction and just make a storage_backend_readonly (if, for example, it can only be on one storage backend)